eng/mgmt/automation/generate.py

```diff
@@ -48,7 +48,11 @@ def generate(
         module,
     )
     shutil.rmtree(os.path.join(output_dir, 'src/main'), ignore_errors = True)
-    shutil.rmtree(os.path.join(output_dir, 'src/samples'), ignore_errors = True)
+    if os.path.exists(os.path.join(output_dir, 'src/samples/README.md')):
+        # samples contains hand-written code
+        shutil.rmtree(os.path.join(output_dir, 'src/samples/java', namespace.replace('.', '/'), 'generated'), ignore_errors = True)
+    else:
+        shutil.rmtree(os.path.join(output_dir, 'src/samples'), ignore_errors = True)
```

Comment:
If no hand-written sample, the sample folder path will not contain /generated?

Reply:
If no hand-written sample, we just delete the `samples` folder, delete everything.

Reply:
If we:
1. generate sample under samples/
2. add manual samples
will the generated sample be updated in the future ?

Reply:
Firstly we would almost never add manual sample to Lite SDK.

Secondly, if we do add manual sample and see that folder already have generated samples, the best approach maybe doing a regen first, then add manual sample. Or one can move them manually. Any case, this probably not going to happen.

Reply:
ok, just want to make sure manual sample will not be deleted.